src/pages/home.jsx

```diff
@@ -1,21 +1,69 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { H1 } from '../components/atm.typography/typography.styled'
 import { FieldButton } from '../components/mol.field-button/field-button';
 import { SearchField } from '../components/mol.search-field/search-field';
+import { NavAccountLinks } from '../components/mol.menu/menu.styled';
+import { Button } from '../components/atm.button/button';
+import { HBox } from '../components/mol.box/box.styled';
+import { Link } from 'react-router-dom';
 
 
 export const Home = () => {
     const [input, setInput] = useState({});
+    const [quickMenu, setQuickMenu] = useState([]);
```

Comment:
esse `quickMenu` eh algo q pode mudar? se for lista fixa podemos criar um `const` nesse arquivo e usar, em vez de ter hook